x-pack/plugins/reporting/server/export_types/csv/types.d.ts

```diff
@@ -8,16 +8,6 @@ import { BaseParams, BasePayload } from '../../types';
 
 export type RawValue = string | object | null | undefined;
 
-interface DocValueField {
```

Comment:
These two were not used anywhere

Reply:
Would be nice if we could add lint rules or something similar for it. I bet there's a few others around